config/initializers/rollbar.rb

```diff
@@ -0,0 +1,51 @@
+require 'rollbar/rails'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.